### PR TITLE
attempted to update,

### PR DESCRIPTION
**Pull request by J9921**

### CODE REVIEW

**Comment by J9921** [comment]
> Please show screenshots of it working

Pages are working, although the side is currently being modified and due to launch publicly shortly, Only few staff/managers have access right now. But here's the screenshots with the pages including urls






createnews
newacc
rules
allarticles
editaccounts
bookaway

File: Bounce/dist/presence.js [hunk 1]
```diff
@@ -117,8 +117,30 @@ presence.on("UpdateData", () => {
...
+            presenceData.details = "Creating Article"
+         break;
+         case "editor.allstories":
+            presenceData.details = "Viewing Articles"
+         break;
+         case "manager.modifyaccounts":
+            presenceData.details = "Updating Account"
+         break;
+         case "manager.createaccount":
+            presenceData.details = "Creating Account"
```
Comment (Bas950) [inline Bounce/dist/presence.js]: Where is the switch? xD

Comment (J9921) [reply Bounce/dist/presence.js]: Tbh, Our Devs away so it'll take me a while to figure out what ya mean by that lool

Comment (J9921) [reply Bounce/dist/presence.js]: I'm assuming it's something like? ' switch (document.location.hash.replace("/portal/", "").replace("#", "")) { '

File: Bounce/dist/presence.js [hunk 2]
```diff
@@ -117,8 +117,34 @@ presence.on("UpdateData", () => {
     }
      else if(document.location.pathname.startsWith("/portal")) {
       presenceData.state = "Browsing Staff Panel";
-      presenceData.details = "Logging in"
-  }
+      presenceData.details = "Logging in"
```
Comment (Bas950) [inline Bounce/dist/presence.js]: maybe add logging in as a default in the switch?

Comment (doomlerd) [reply Bounce/dist/presence.js]: default is not required.

**Comment by doomlerd** [comment]
@J9921 we'll have to close the PR in 3 days, you still haven't provided a screenshot of the changes working ( in Discord ) so again, please provide screenshots of it working.

**Comment by J9921** [comment]
> @J9921 we'll have to close the PR in 3 days, you still haven't provided a screenshot of the changes working ( in Discord ) so again, please provide screenshots of it working.

image

**Pull request merged**